### PR TITLE
Add Matter Geneva blog

### PR DESCRIPTION
**Pull request by GuySie**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+And all this effort isn’t just aimed at the implementation of the Matter standard itself, we’re also hard at work to make using Matter as easy as possible for you. We’re designing UI changes in Home Assistant that will make it far easier to understand Matter devices, from the onboarding process to diagnosing issues to understanding your Thread networks.
+
+## Matter live stream
+As you can imagine, there’s a lot happening around Matter right now, and we want to tell you about all the things that we’re working on in Home Assistant! That’s why, in January 2024, we’re going to have a live stream where we will give you a full update on the state of Matter and go in-depth about what we have planned. We’ll announce the date soon.
+
+But we don’t want it to be just a stream of us talking Matter to you. We know you have many questions about Matter, things that are unclear, or that you want our help to understand. So help us make the stream even better, and please send us your burning questions in the comments below!
```
Comment (balloob) [inline source/_posts/2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown]:
```suggestion
TL;DR: We represented Home Assistant, our community, and the Open Home vision at the Matter member meeting in Geneva. We’re hosting a live stream to talk Matter in January to update you about our progress and answer your questions. Leave your questions in the comments below!

Two weeks ago me, Marcel van der Veldt, and Stefan Agner, traveled to Geneva to represent Home Assistant, our community and the Open Home Vision at the Member Meeting of the Connectivity Standards Alliance (CSA). This is an important meeting where companies from all over the world meet to talk and decide about the Matter standard and how to implement it.

_(Matter is the new smart home standard that promises to make everyone’s smart home devices work with each other across platforms and ecosystems, locally and privately. It’s being developed by the CSA, which is also responsible for Zigbee)._

<p class='img'>
<img src="/images/blog/2023-11-matter-member-meeting/matter-image.jpg">
Stefan and Marcel
</p>

We were able to attend because [Nabu Casa](https://www.nabucasa.com) is a member of the CSA. We pay for this with the revenue from your Home Assistant Cloud subscriptions (thank you!). CSA membership ensures that we have access to official technical documentation and support to build Matter into Home Assistant. It also gives us a voice inside the CSA, which we use to advocate for the interests of Home Assistant users and our Open Home vision.

<!--more-->
We love to attend these events and hear amazing stories from the people we meet. When talking with engineers working on Matter in the CSA, we noticed a lot of them run Home Assistant at their own homes and are enthusiastic supporters of the project! Talking to device manufacturers made us realize that Home Assistant is used as a test platform because our Matter server is the most flexible. This is great because it increases the likelihood that Matter devices work great in Home Assistant, as devices will have been implemented and tested on it!

Our participation in the development of Matter also means we’re in touch with engineers at other companies on a regular basis. One of the most popular Matter devices on the market right now, the Eve Energy Plug, has custom support for energy metering - a function not yet supported in the current Matter standard. The only way to see your energy usage was inside Eve’s own app. We’ve worked together with Eve so we can now read the measurements from their plugs via custom Matter clusters and show them in Home Assistant! This feature will be available in the December release of Home Assistant. If you own Eve Energy Plugs, this is our early Christmas present for you.

## Home Assistant and Matter

We haven’t just joined the CSA; we also have dedicated developers working on making sure that Matter is a first-class citizen within Home Assistant. In our eyes, it’s ideal for Home Assistant users because of the local nature of the protocol. It is our goal to make Home Assistant an official certified Matter controller, registered with the CSA.

To talk more about our ongoing progress with Matter, we’ll be hosting a live stream in January. It will contain a full update on the state of Matter and go in-depth about what we have planned. We’ll announce the date soon.

We don’t want it to be just a stream of us talking Matter to you. We know you have many questions about Matter, things that are unclear, or that you want our help to understand. So please send us your burning questions in the comments below!
```

**Review by c0ffeeca7 — DISMISSED** [review]
LGTM, @GuySie :+1:

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @GuySie :+1:
